src/com/base/game/combat/SpecialAttack.java

```diff
@@ -997,6 +997,61 @@ public boolean isConditionsMet(GameCharacter owner) {
 		}
 	},
 
+	COW_GORE(50, "Gore", "HornIcon", Colour.DAMAGE_TYPE_PHYSICAL, DamageType.PHYSICAL, DamageLevel.HIGH, DamageVariance.LOW, SpecialAttackSpellCosts.MEDIUM, Util.newHashMapOfValues(new Value<StatusEffect, Integer>(StatusEffect.CRIPPLE, 4))) {
+		@Override
+		public String applyEffect(GameCharacter caster, GameCharacter target, boolean isHit, boolean isCritical) {
+
+			float damage = calculateDamage(caster, target, isCritical), cost = calculateCost(caster);
+
+			descriptionSB = new StringBuilder();
+			
+			if (caster == Main.game.getPlayer()) {
+				descriptionSB.append(UtilText.genderParsing(target,
+						"<p>" + "With a burst of energy, you leap forwards, trying to ram your horns at " + target.getName("the") + "."
+								+ (isHit ? " Your cow-like horns slam into " + target.getName("the") + "'s body,"
+										+ " and you manage to cause some serious damage with your sharp horns before <she> manages to throw you off of <herPro>."
+										: target.getName("The") + " manages to jump to one side, and there's an audible woosh as your move through the thin air.")
```

Comment:
"and there's an audible woosh as your move through the thin air."

"as your move" -> "as you move"